### ISSUE TITLE
cosmetic:  version of DSM and vDSM is 7.1 not 7.2

### ISSUE THREAD

**Issue by GuitarBilly** [kroese/virtual-dsm#3]
hello, i tried your virtual-dsm and it is working very nice.

One item i would like to note is that the version of DSM and vDSM is not 7.2 as mentioned in the readme.md
`A docker container of Synology DSM v7.2 using KVM.`

but version 7.1 shows on the big blue welcome screen and DSM 7.1.1-42962 in the InfoCenter respectively.

This also matches the latest version of vDSM i can find from synology, 7.1.1:
https://www.synology.com/en-us/support/download/VirtualDSM

**Comment by QuadNL** [comment]
Indeed, but updating to 7.2 beta is poss, did it yesterday without issues.
